.github/workflows/release.yml

```diff
@@ -28,3 +28,30 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  trigger-image-build:
+    runs-on: ubuntu-latest
+    needs: release
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set ref
+        id: set-ref
+        run: ::set-output name=tag::${GITHUB_REF#refs/*/}
+
+      - name: Trigger pke-image build
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            await github.rest.actions.createWorkflowDispatch({
+              owner: 'banzaicloud',
+              repo: 'pke-image',
+              workflow_id: 'packer-build.yml',
+              ref: 'master',
```

Comment:
sure?

Reply:
Yes, This will trigger the packer-build.yml of pke-image on master branch.